picard/solver.py

```diff
@@ -121,7 +130,18 @@ def picard(X, fun='tanh', n_components=None, ortho=True, whiten=True,
         Number of iterations taken to converge. This is
         returned only when return_n_iter is set to `True`.
     """
+    np.seterr(all='raise')
```

Comment:
this is a global change of numpy state. It's bad. You don't want to change behavior of all packages that use picard